packages/smooth_app/lib/database/local_database.dart

```diff
@@ -26,6 +27,7 @@ class LocalDatabase extends ChangeNotifier {
       DaoProductList(localDatabase),
       DaoStringList(localDatabase),
       DaoString(localDatabase),
+      DaoInt(localDatabase),
```

Comment:
This comment has nothing to do there but github tells me that the `analytics_helper.dart` has so many changes that the differences are not displayed.

Comments about `analytics_helper.dart`:

`const AnalyticsHelper();`: that is useless as we only use static methods.

`_trackConstructor` is by far the lousiest method I've seen in the 21st century.
You say that it's a copy of `package:matomo_forever`:
1. why don't you just use that package?
2. if you do need to copy that code, please put it in a dedicated `matomo_justified_copy.dart` file or something like that, with appropriate comments.

`_getPreviousVisitUnix` and `_getFirstVisitUnix`: using `int` instead of `String` would make sense

`_getAppVisits`, `_getPreviousVisitUnix` and `_getFirstVisitUnix` are all `static` and all of them have only one parameter: `LocalDatabase`.
That should probably make sense to create a new class with one field (a `LocalDatabase`) and those 3 methods as non `static`.